addon/lib/document.js

```diff
@@ -121,6 +121,10 @@ export default Base.extend({
       return Array.isArray(arr) ? arr.map((obj) => obj[key]) : null;
     });
     documentJexl.addBinaryOp("intersects", 20, intersects);
+    documentJexl.addTransform("debug", (any) => {
+      console.log("Jexl Debug: ", any);
```

Comment:
`console.debug` might be the better choice here

Reply:
Good idea!

Reply:
A little bit of context (e.g the question slug) would be nice

Reply:
Is there a way to get at the question slug?

Reply:
Just checked - sadly no. So ignore my comment ^^

Reply:
Would have been nice though :smiley: